resources/views/layouts/app.blade.php

```diff
@@ -139,6 +139,8 @@
 
                   @if (\Auth::user()->isHeadMaster())
                     <li><a  href="{{ route('dashboard') }}" class="nav-item is-tab"><span class="icon"><i class="fa fa-bar-chart"></i></span> Dashboard</a></li>
+                    <li><a href="{{ route('news') }}" class="nav-item is-tab"><span class="icon"><i class="fa fa-newspaper-o"></i></span>News</a></li>
```

Comment:
Question: you are going directly to news, but don't you need to go to dashboard first?